lib/core/src/tab/tab.tsx

```diff
@@ -28,14 +28,36 @@ interface TabStyle {
 }
 
 interface Props {
+	/**
+	 * Title and Body of each tab.
+	 */
 	children: Tab[];
+	/**
+	 * Define if tabs padding is removed.
+	 */
```

Comment:
This good, but usually when we "remove" something we should say a little bit that it is existed.. like "Tab's panels have padding out of the box. Use this prop to remove the padding"